### PR TITLE
Refactor

### PR DESCRIPTION
**Pull request by rafal-gorecki**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,33 +1,45 @@
...
 ```
 
-# On PC
-### Open Foxglove application in browser:
-Open web browser and go to `<ROSbot_ip>:8080` website. Chrome/Chromium browser is highly recommended.
+**4. Open Foxglove application in browser.**
+
+To open Foxglove type `<ROSBOT_IP/HUSARNET_NAME>:8080` in your browser search bar. 
+
+:::info
```
Comment (delihus) [inline README.md:29]:
```suggestion
```

Comment (delihus) [inline README.md]:
```suggestion
> [!NOTE]
```

File: README.md [hunk 2]
```diff
@@ -1,33 +1,45 @@
...
+You should use **Chrome/Chromium** browser.
```
Comment (delihus) [inline README.md]:
```suggestion
> You should use **Chrome/Chromium** browser.
```

File: README.md [hunk 3]
```diff
@@ -1,33 +1,45 @@
...
+:::
```
Comment (delihus) [inline README.md]:
```suggestion
```

**Review by delihus — CHANGES_REQUESTED** [review]
The `:::info` doesn't render on Github. Maybe Note will be good. The rest is great!

image

For e.g.
image

**Comment by delihus** [comment]
Please add info how to flash the correct version of firmware from `husarion/rosbot-xl:humble-0.8.2-20230913`.

**Review by delihus — APPROVED** [review]
Thanks!

**Review by JanBrzyk — DISMISSED** [review]

**Review by JanBrzyk — APPROVED** [review]

**Review by JanBrzyk — APPROVED** [review]

**Pull request merged**